sats-dna/src/main/kotlin/com/sats/dna/components/SatsTabRow.kt

```diff
@@ -0,0 +1,93 @@
+package com.sats.dna.components
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.TabRowDefaults
+import androidx.compose.material3.TabRowDefaults.tabIndicatorOffset
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import com.sats.dna.LocalUseMaterial3
+import com.sats.dna.theme.SatsTheme
+
+@Composable
+fun SatsTabRow(
+    selectedTabIndex: Int,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    CompositionLocalProvider(LocalUseMaterial3 provides true) {
```

Comment:
Does it make sense to instead just do this? It should probably always have the surface colors, one would think?

```suggestion
    SatsSurface(useMaterial3 = true) {
```

Reply:
Or maybe it's more correct to make it transparent, like I see you do below. 🤔

Reply:
In the design it doesn't have a set background color afaict. Might be risky if we decide to put it somewhere where the background color doesn't fulfil double AA with the text color here.

[One of the instances where this is used](https://github.com/sats-group/member-app-android/blob/5d459a4cb56caf9d6238595aea8cb6c408065719/features/my-activity/src/main/kotlin/com/sats/features/myactivity/MyActivityTopAppBar.kt#L61), the background color follows that of the top app bar (with transitioning colors when scrolled). Which would not work with a set background color, so we could open this component to accept background colors, but then again we would have the same issue as with a transparent background unless we also open for changing content colours to match the background color which would then deviate from the colors set in the DS 🥴 

Tl;DR: We should maybe keep it transparent unless we want it to have totally customisable background and content colors.

Reply:
I honestly don't know what's best there's no background color set in the DS but there are text / indicator colors. So idk if we should open the component up for both setting a background color and a content color. Or maybe just keep it transparent with fixed text and indicator colors 🤔

Reply:
Yeah, let's keep the background color transparent. That's what seems to be intended in the DS, too.